_posts/2023-09-27-trino-gateway.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Trino Gateway has arrived"
+author: Manfred Moser, Martin Traverso
+image: /assets/images/logos/trino-gateway-small.png
+excerpt_separator: <!--more-->
+show_pagenav: false
+---
+
+You started with one Trino cluster, and your users like the power for SQL and
+[querying all sorts of data sources]({{site_url}}/ecosystem.html#data-sources).
+Then you needed to upgrade and got a cluster for testing going. That was a while
+ago, and now you run a separate cluster configured for ETL workloads with
+fault-tolerant execution, and some others with different configurations.
+
+With Trino Gateway we now have an answer to your users request to provide on URL
+for all the clusters. Trino Gateway has arrived!
+
+<!--more-->
+
+Today, we are happy to announce our [first release of Trino
+Gateway](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-3-26-sep-2023).
+The release is the result of many, many months of effort to move the legacy
+Presto Gateway to Trino, start a refactor of the project, and add numerous new
+features.
+
+Many larger deployments across the Trino community rely on the gateway as a load
+balancer, proxy server, and configurable routing gateway for multiple Trino
+clusters. User don't need to worry about what catalog and data source is
+available in what Trino cluster. Trino Gateway exposes one URL for them all.
+Administrators can ensure routing is correct and use the REST API to configure
+the necessary rules.
+
+Up to now, many users had to maintain separate forks of the legacy Presto
+Gateway. Some of these users created numerous improvements in isolation of each
+other, sometimes even implementing the same feature multiple times. This first
+release of Trino Gateway starts a strong collaboration of some of these users.
+Bloomberg contributed the main bulk of the new features, maintainers and
+contributors from Starburst pulled together the stakeholders and managed the
+project, and other collaborators from Apple, Naver, LinkedIn, Lyft, and Dune are
+already helping out and ready to move the project forward.
+
```

Comment:
are we mentioning the authentication/authorization features here? They are a major ask from users over the presto-gateway?

Reply:
Done